README.md

```diff
@@ -75,7 +74,7 @@ Now you can try `./v run examples/hello_world.v` (`v.exe` on Windows).
 
 V is being constantly updated. To update V, simply run:
 
-```
+```.bash
```

Comment:
Is it `.bash` or `bash` ?

Reply:
It is bash.
fix it.
https://github.com/vlang/v/pull/6055/commits/fbe41dca3a112b769e25fa2945f8d5c6825e899c